plugins/labhub.py

```diff
@@ -73,7 +73,7 @@ def TEAMS(self, new):
 
     # Ignore LineLengthBear, PycodestyleBear
     @re_botcmd(pattern=r'^(?:(?:welcome)|(?:inv)|(?:invite))\s+(?:(?:@?([\w-]+)(?:\s*(?:to)\s+(\w+))?)|(me))$',
-               re_cmd_name_help='invite [to team]')
+               re_cmd_name_help='invite (@<username> to <team>|me)')
```

Comment:
shouldn't it be `[]` instead of `()`, as it's still optional?

Reply:
And btw, the `@` is also optional :)

(You could actually put the addition of `[@]<username>` and `<team>` into a separate commit, in fact that's a "different" issue :D)

Reply:
Umm, it's optional in the sense that it should be exactly one either @username to team or me.
![untitled3](https://user-images.githubusercontent.com/20523845/36682645-a2917efc-1b41-11e8-9c84-32c90beeeef2.png)

Reply:
Agreed, '@' will also be optional :)